README.md

```diff
@@ -33,59 +35,45 @@ This is the second option.
 
 ## Third thought, docx
 
-But I also needed that the many time (most of the time) when I have to clean up the mess of multiple internal edits with clients working in docx. Therefore I have added the same structure (also OOXML is an XML bunch of files zipped together) for that file, without the need to convert them in ODT as I used to do before.
+But I also needed that the many time (most of the time) when I have to clean up the mess of multiple internal edits with clients working in docx. Therefore I have added the same functions (also OOXML is a bunch of XML, files zipped together, just a bit more clumsy) for that file, without the need to convert them in ODT as I used to do before (round robin tends to screw up).
 
-So it will also work with docx (OOXML text document), although MS Word® has a feature to change data as an afterthought (but all or none, AFAICR) and this script might not be strictly necessary.
+So it will also work with docx (OOXML text document), although MS Word® has a feature to change data as an afterthought (but all or none, AFAICT) and this script might not be strictly necessary (I think it is).
 
 ## Fourth thought, python3
 
-I have attempted a porting to python3, actually everything has been rewritten with some more sense. It should work for any operating system, as I have abstracted the paths via the `os` library. Hopefully.
-
-# HOWTO BASH
+Since scripting is limited, I decided to port everything to python3, actually everything has been rewritten with some more sense. It should work for any operating system, as I have abstracted the paths via the `os` library. Hopefully. See [Windows and Mac] for more instructions.
 
-Requires a recently updated version of Linux (I target bash 4.0, roughly), preferably git (but it's not necessary) and a working knowledge of running scripts. The easiest way is to clone the repository somewhere and point to the script from the command line.
 
-This is still work in progress, but it's now reasonably good to go.
+# HOWTO python
 
-Suppose that you have installed (copied) it in `~/scripts/anonymize.sh` and that your documenti is in `~/documents/doc.odt`
+It should be sufficient to have a running python3 environment, the imported libraries are all from the standard set.
 
-Do this:
 
-```shell
-ls -l ~/scripts/anonymize.sh # (make sure it's "executable")
 ```
+[script_directory]/anonymize.py filename.[odt|docx]
 
```

Comment:
There is an extra newline here.

Reply:
I meant the newline right after `[script_directory]/anonymize.py filename.[odt|docx]`, which looks strange when rendered:

![image](https://user-images.githubusercontent.com/8166212/172422591-a1d84bb0-bc27-410f-b52e-9518d6c584e1.png)